command/push.go

```diff
@@ -306,6 +371,7 @@ func (c *atlasPushClient) Get(name string) (map[string]interface{}, error) {
 
 	var variables map[string]interface{}
 	if version != nil {
+		// TODO: merge variables and TFVars
 		//variables = version.Variables
```

Comment:
Oh, I forgot about this. I'm not sure if this is supposed to still be commented out or not :/